resources/arduino_files/libraries/dallas-temperature-control/DallasTemperature.h

```diff
@@ -0,0 +1,242 @@
+#ifndef DallasTemperature_h
+#define DallasTemperature_h
+
+#define DALLASTEMPLIBVERSION "3.7.2"
```

Comment:
I notice that "3.7.9" is the latest for this. Any reason to start with an old one here? I realize the features probably haven't changed much, just wondering.

Reply:
Just what I had already been using. Since I'm not doing extensive testing of this, I'd prefer to hold off changes until able to do so.